Functions/xASL_io_SplitASL.m

```diff
@@ -218,6 +235,30 @@ function xASL_io_SplitASL(inPath, iM0, iDummy)
 
 end
 
+%% xASL_io_SplitASL_PostModify
+function [jsonM0, jsonASL] = xASL_io_SplitASL_PostModify(ASLname, jsonM0, jsonASL, iM0, iDummy)
+
+    % Check that both structs are not empty dummy structs
+    if ~isempty(fieldnames(jsonASL)) && ~isempty(fieldnames(jsonM0))
+        
+        % Add IntendedFor field
+        [~, file, extension] = xASL_fileparts(ASLname);
+        jsonM0.IntendedFor = ['perf/' file extension];
+        
+        % Remove M0Type field of M0 JSON if it still exists
+        if isfield(jsonM0, 'M0Type')
+            jsonM0 = rmfield(jsonM0, 'M0Type');
+        end
+
+        % Change M0Type field of ASL JSON to Separate
+        if isfield(jsonASL, 'M0Type') && (~isempty(iM0) || ~isempty(iDummy))
```

Comment:
Not sure why it is relevant to have the iDummy - these are about excluding stuff from ASL, not related to M0.

The ~isempty(iM0) probably isn't necessary either. Because if we are running splitting and there's a jsonM0, then it is clear that we have split the files and M0.nii is there - though I'm not 100% sure about this one and it doesn't hurt to have it here (unlike the iDummy though).

Reply:
Ah perfect, that is the part where I wasn't 100% sure if it's correct.
In line 78 we check `if isempty(iM0) && isempty(iDummy)` anyway, so this part is redundant.
I just wanted to make sure, that we either split M0 or Dummy scans, so that if someone would call it and there was not M0 within the sequence, that my code section wouldn't lead to unexpected bugs. We can definitely get rid of the iDummy part.

Reply:
Yes - without iM0 or iDummy, there's nothing to split. But if there's only iDummy, then we throw some volumes away, but nothing changes on the way M0 is defined. It will mostly be `separate` anyway, but there's still a chance that the M0 will be defined by a single value. So if the only reason for calling the split were dummy scans, we don't want to change the M0Type. So iDummy indeed hasto go